### PR TITLE
Feature: Union schema compatibility

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
Issue: #21 

Confirm the following files were correctly updated automatically:
- [x] CHANGELOG
- [x] README
- [x] stg_apple_search_ads.yml, src_apple_search_ads.yml, apple_search_ads.yml (depending if source or transform)
- [x] docs.md
- [x] joins
- [ ] ~window functions (partition by)~
- [x] 'source_relation' column added in staging and transform models
- [x] tests

Manual updates:
- [x] Add `source_relation` to downstream models if necessary
- [x] Finish any incomplete/incorrect joins/partitions
- [x] Update tests to include `source_relation` in unique-combination-of-cols if necessary
    - May need to remove some uniqueness tests in favor of the combo test
    
Validation:
- [x] `dbt run` locally passes
- [x] `dbt test` locally passes

### CODE REVIEW

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:4]: Actually, can we remove the README "hide" sections in these updates. This way we are able to link directly to the section in the README where the users needs to understand how to configure the union feature.

Otherwise, I worry this cool feature is hidden from people actually using it.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
These updates look great! I have one small comment about updating the README and CHANGELOG. Once those are applied then this should be ready to merge once the others are all approved.

**Comment by AaronCoquet-Easypark** [comment]
When should we expect this to be merged? Is there anything left to be done?

**Comment by fivetran-catfritz** [comment]
Hi @AaronCoquet-Easypark we are going through final validations for this package and its integration into our rollup `ad_reporting` package. We are planning to release this in the next week or two.

**Pull request merged**
